### PR TITLE
change to MIT license

### PR DESCRIPTION
**Pull request by oluwatobiBolu**
kindly review
deleted the previous license- it doesn't allow for script reusing
changed to MIT license for infrastructure/scripting as per the issue #421

### CODE REVIEW

**Comment by oluwatobiBolu** [comment]
> kindly review deleted the previous license- it doesn't allow for script reusing changed to MIT license for infrastructure/scripting as per the issue #421

my name is Oluwatobi Adelakun, an outreachy applicant, please kindly review the pull request and await your feedback

**Review by oluwatobiBolu — COMMENTED** [review]
I was able to remove the green links 
made it bold and still maintain its hovering properties as a link
please kindly review
apprecitae your feedback

**Comment by bebatut** [comment]
Thanks @oluwatobiBolu for this contribution-
It is usually better to have a Pull Request for 1 issue. Could you update this Pull Request to focus only on the License?
Did you make sure that CC-BY-SA 4.0 (for content) is mentioned on the README, and on the website?

**Comment by oluwatobiBolu** [comment]
> Thanks @oluwatobiBolu for this contribution- It is usually better to have a Pull Request for 1 issue. Could you update this Pull Request to focus only on the License? Did you make sure that CC-BY-SA 4.0 (for content) is mentioned on the README, and on the website?

i have reverted the last commit so the pull request focuses on just the license issue
thanks for the feedback

File: README.md [hunk 1]
```diff
@@ -436,4 +437,434 @@ To order them:
       -c <cohort id> \
    ```
 
-3. Update `_config.yml` file to add new cohort in collection
\ No newline at end of file
+3. Update `_config.yml` file to add new cohort in collection
+
+## License
+
+Attribution-ShareAlike 4.0 International
```
Comment (bebatut) [inline README.md:444]: We do not the full license here in the README file. Just a small bit like in the footer of the website

Comment (oluwatobiBolu) [reply README.md:444]: I understand now 
i added a small bit but it was in the middle.
i will carry out changes now

**Review by oluwatobiBolu — COMMENTED** [review]
i think this is appropriate.

**Comment by oluwatobiBolu** [comment]
> i think this is appropriate.

what do you think?
i didn't necessarily have to write all the conditions.
does the summary cover it?

File: README.md [hunk 2]
```diff
@@ -436,4 +437,23 @@ To order them:
       -c <cohort id> \
    ```
 
-3. Update `_config.yml` file to add new cohort in collection
\ No newline at end of file
+3. Update `_config.yml` file to add new cohort in collection
+
+## License
+
+Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
```
Comment (bebatut) [inline README.md:444]: We should write more clearly what is going under which license.
Something like here: https://github.com/galaxyproject/training-material/#license

**Comment by bebatut** [comment]
There is a conflict there. Can you resolve it?

File: README.md [hunk 3]
```diff
@@ -436,4 +437,13 @@ To order them:
...
+
+The content of this website are licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
+
+
```
Comment (bebatut) [inline README.md:448]: The information about MIT license got lost

File: README.md [hunk 4]
```diff
@@ -436,4 +437,13 @@ To order them:
       -c <cohort id> \
    ```
 
-3. Update `_config.yml` file to add new cohort in collection
\ No newline at end of file
+3. Update `_config.yml` file to add new cohort in collection
+
+## License
+
+Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
```
Comment (bebatut) [inline README.md:444]: Sentence not needed

File: .gitignore [hunk 5]
```diff
@@ -1,5 +1,5 @@
-_site
```
Comment (bebatut) [inline .gitignore:1]: Could you remove the changes on this file, please?

**Comment by bebatut** [comment]
GitHub still says there is a conflict on the `README` file. Could you try to resolve it? Thanks

File: LICENSE.md [hunk 6]
```diff
@@ -1,427 +1,21 @@
...
-use of the trademark "Creative Commons" or any other trademark or logo
-of Creative Commons without its prior written consent including,
-without limitation, in connection with any unauthorized modifications
-to any of its public licenses or any other arrangements,
-understandings, or agreements concerning use of licensed material. For
-the avoidance of doubt, this paragraph does not form part of the
-public licenses.
-
-Creative Commons may be contacted at creativecommons.org.
+MIT License
```
Comment (Jolah1) [inline LICENSE.md:1]: The License file should not have an extension ".md"

**Review by Jolah1 — COMMENTED** [review]
I think the License file should not have an extention ".md"

**Comment by bebatut** [comment]
Thanks @oluwatobiBolu for fixing the conflict.
I am thinking about something: could you also change the footer in the website (in the `_includes` folder) to add a sentence similar to the one in the README file about the infrastructure License? That would really clarify what people can do. Thanks

File: _includes/default-footer.html [hunk 7]
```diff
@@ -39,6 +39,9 @@ <h3>Get in touch</h3>
             The website content is licensed
             <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY SA 4.0</a>.
           </p>
+          <p>
```
Comment (bebatut) [inline _includes/default-footer.html:42]:
```suggestion
```

File: _includes/default-footer.html [hunk 8]
```diff
@@ -38,6 +38,7 @@ <h3>Get in touch</h3>
           <p>
             The website content is licensed
             <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY SA 4.0</a>.
+            The code behind the infrastructure is licensed under the <a href="https://github.com/open-life-science/open-life-science.github.io/blob/main/LICENSE.md">MIT License</a>
```
Comment (bebatut) [inline _includes/default-footer.html:41]:
```suggestion
            The code behind the infrastructure is licensed under the <a href="{{ site. repository }}/blob/main/LICENSE.md">MIT License</a>
```

File: _includes/default-footer.html [hunk 9]
```diff
@@ -38,6 +38,7 @@ <h3>Get in touch</h3>
           <p>
             The website content is licensed
             <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY SA 4.0</a>.
+            The code behind the infrastructure is licensed under the <a href="{{ site. repository }}/blob/main/LICENSE.md">MIT License</a>
```
Comment (bebatut) [inline _includes/default-footer.html:41]:
```suggestion
            The code behind the infrastructure is licensed under the <a href="{{ site.repository }}/blob/main/LICENSE.md">MIT License</a>
```

File: _includes/default-footer.html [hunk 10]
```diff
@@ -38,6 +38,7 @@ <h3>Get in touch</h3>
           <p>
             The website content is licensed
             <a href="http://creativecommons.org/licenses/by-sa/4.0/">CC BY SA 4.0</a>.
+            The code behind the infrastructure is licensed under the <a href="{{ site.repository }}/blob/main/LICENSE.md">MIT License</a>
```
Comment (bebatut) [inline _includes/default-footer.html:41]:
```suggestion
            The code behind the infrastructure is licensed under the <a href="{{ site.github.repository_url }}/blob/main/LICENSE.md">MIT License</a>
```

**Comment by bebatut** [comment]
It is a Jekyll plugin: https://jekyll.github.io/github-metadata/

**Review by yochannah — APPROVED** [review]

**Review by emmyft — APPROVED** [review]

**Comment by oluwatobiBolu** [comment]
> It is a Jekyll plugin: https://jekyll.github.io/github-metadata/

thanks for the resource

**Comment by oluwatobiBolu** [comment]
@bebatut  I think there are no more changes to be made here..and hopefully can be merged

**Comment by bebatut** [comment]
We just need the approval from @malvikasharan there. But she is currently in holidays

**Comment by AndreaSanchezTapia** [comment]
Hi @malvikasharan just tagging you here for your approval :)

**Review by malvikasharan — APPROVED** [review]

**Comment by malvikasharan** [comment]
So late to this! Apologies all. I had not been previously oinged until recently @bebatut brought this to my attention. I have approved this and merging this.

**Pull request merged**